### DISCUSSION TITLE
resource scroll, arrow pointing to green number after it is already fulfilled

### DISCUSSION THREAD

**Discussion by dvdiscool** [ldtteam/minecolonies#8226] [Q&A]
![2022-04-06_19 38 07](https://user-images.githubusercontent.com/102939530/161921848-7bfc9b6f-d7ee-4cfb-b825-2116c52988e7.png)
I was just wondering what the box and arrow pointing to the green text are for, it has only happened for builders hut 2 and now builders hut 3 both sandstone

**Comment by Raycoms** [comment]
indicates still an incoming delivery

**Reply by dvdiscool** [reply]
cool! thanks
